docker/.env

```diff
@@ -35,8 +36,8 @@ WHATRECORD_SERVER_SCAN_PERIOD=600
 WHATRECORD_AUTOSAVE_RELOAD_PERIOD=60
 
 # These are optional to show off the plugins:
-HAPPI_CFG=/usr/share/whatrecord/support/happi.cfg
+WHATRECORD_HAPPI_CFG=/usr/share/whatrecord/support/happi.cfg
```

Comment:
Is the docker container supposed to come with a happi database to view?  mine fails to load

<details>

```
2023-05-30 23:33:55,595 - PID 49            util.py: 91  run_script_with_json_output WARNING  Standard error output while running script ('"/usr/local/bin/python" -m whatrecord.plugins.happi'):
    ! Traceback (most recent call last):
    !   File "/usr/local/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    !     return _run_code(code, main_globals, None,
    !   File "/usr/local/lib/python3.10/runpy.py", line 86, in _run_code
    !     exec(code, run_globals)
    !   File "/usr/local/src/dev-whatrecord/whatrecord/plugins/happi.py", line 423, in <module>
    !     _cli_main()
    !   File "/usr/local/src/dev-whatrecord/whatrecord/plugins/happi.py", line 416, in _cli_main
    !     results = main(**vars(args))
    !   File "/usr/local/src/dev-whatrecord/whatrecord/plugins/util.py", line 35, in wrapper
    !     results = func(*args, **kwargs)
    !   File "/usr/local/src/dev-whatrecord/whatrecord/plugins/happi.py", line 361, in main
    !     client = happi.Client.from_config()
    !   File "/usr/local/lib/python3.10/site-packages/happi/client.py", line 794, in from_config
    !     cfg = cls.find_config()
    !   File "/usr/local/lib/python3.10/site-packages/happi/client.py", line 906, in find_config
    !     raise OSError("No happi configuration file found. Check HAPPI_CFG.")
    ! OSError: No happi configuration file found. Check HAPPI_CFG.
```
</details>

Reply:
Hmm, good catch. It should:
https://github.com/klauer/whatrecord/blob/8342ff4cb93cac8a0e250942b0cc2c58a37a3999/docker/support/happi_db.json#L1

Oh, that's probably because I stupidly changed the `HAPPI_CFG` env var which happi relies on to `WHATRECORD_*`

Reply:
Should be back in business:
<img width="1781" alt="image" src="https://github.com/pcdshub/whatrecord/assets/5139267/944ddd2c-8368-4498-8f09-7420b5881b18">